platforms/web/src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -49,26 +61,31 @@ export default function AppRoutes() {
   }
 
   const shouldManageProfiles =
-    !!user && profilesEnabled && !profile && (accessModel === 'SVOD' || accessModel === 'AUTHVOD') && !userModal && !location.pathname.includes('/u/profiles');
+    !!user &&
+    profilesEnabled &&
+    !profile &&
+    (accessModel === 'SVOD' || accessModel === 'AUTHVOD') &&
+    !userModal &&
+    !location.pathname.includes(PATH_USER_PROFILES);
```

Comment:
Wow, so many things to consider.

Reply:
Yes... 😅 But the condition hasn't changed; it's only formatted by Prettier due to the `PATH_USER_PROFILES` usage.